testsuite/MDAnalysisTests/analysis/test_gnm.py

```diff
@@ -44,7 +44,8 @@ def universe():
 def test_gnm(universe, tmpdir):
     output = os.path.join(str(tmpdir), 'output.txt')
     gnm = mda.analysis.gnm.GNMAnalysis(universe, ReportVector=output)
-    gnm.run()
+    with pytest.warns(DeprecationWarning):
```

Comment:
Could you:

a) Decouple these tests from existing tests? It's more of a stylistic comment tbh, but would be a lot easier to work out failures that way. I'm willing to bend on that though if there's a benefit to doing it this way.

b) add a `match` to the warning string on these call (we have a lot of deprecations running about in 1.1, we need to make sure we're picking up the right ones).